lib/utils/resolve-packages.js

```diff
@@ -41,10 +43,33 @@ function resolvePackages(pkg) {
 
 	let packageLocations = [];
 
+	function invokeGeneratorIfReady() {
+		if (packageLocations.length === pkg.length)
+			return creator(packageLocations);
+	}
+
 	pkg.forEach(addon => {
+		// Resolve paths to modules on local filesystem
+		if (fs.existsSync(addon)) {
+			let absolutePath = addon;
+
+			try {
+				absolutePath = path.resolve(process.cwd(), addon);
+				require.resolve(absolutePath);
+				packageLocations.push(absolutePath);
+			} catch (err) {
+				console.log(`Cannot find a valid npm module at ${absolutePath}.`);
```

Comment:
Sorry if this was asked before, why we dont `throw` when provided module wasnt found?

Reply:
Good point—I think that's just a mistake on my part. I will use the code in the `catch` below to log the error and exit the process if we can't resolve the package.